Dockerfile

```diff
@@ -25,14 +25,21 @@ LABEL org.label-schema.build-date=$BUILD_DATE \
 WORKDIR /app
 
 # Dependency for confluent-kafka
-RUN apk --update add librdkafka librdkafka-dev && ldconfig /usr/lib
+RUN apk --update add build-base git cmake bash
+RUN git clone --branch v2.6.1 https://github.com/confluentinc/librdkafka.git /tmp/librdkafka
+RUN cd /tmp/librdkafka && \
+    ./configure && \
+    make && \
+    make install && \
+    ldconfig /usr/lib
```

Comment:
Yikes, why do we need to build from source?

Reply:
Because RUN apk --update add librdkafka only gives you librdkafka 2.4, which is not compatible with confluent-kafka 2.6.1.

Reply:
Ok, so we might be able to install from a repo if we upgrade the docker image in the future